pkg/hub/controllers/cluster/reconciler.go

```diff
@@ -46,12 +46,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 
 	log.Info("got cluster CR from hub", "cluster", cr)
 
-	// Post NodeIP and GeoLocation info only on first run or if the reconciler wasn't run for a while
-	if cr.Status.ClusterHealth == nil || time.Since(cr.Status.ClusterHealth.LastUpdated.Time) > 3*time.Minute {
-		log.Info("updating cluster info on controller")
-		if err := r.updateClusterInfo(ctx, cr); err != nil {
-			log.Error(err, "unable to update cluster info")
-		}
+	// Post NodeIP and GeoLocation info
+	log.Info("updating cluster info on controller")
```

Comment:
The above block and the node controller were already removed by the PR  from @YachikaRalhan in https://github.com/kubeslice/worker-operator/pull/193. Maybe we don't need this PR if those changes are taken from master. Please take a look